modules/shadertools/src/modules/pbr/pbr-fragment.glsl.js

```diff
@@ -41,6 +41,10 @@ uniform sampler2D u_OcclusionSampler;
 uniform float u_OcclusionStrength;
 #endif
 
+#ifdef ALPHA_CUTOFF
```

Comment:
Nit: This feature just adds a single if statement, looks rather cheap, is it really justified to require a shader recompile to change it?